src/deadline/client/cli/_groups/job_group.py

```diff
@@ -608,11 +608,10 @@ def _assert_valid_path(path: str) -> None:
         ],
         case_sensitive=False,
     ),
-    help="The resolution method to use when a file already exists."
-    "Please choose one from the following options. If it is not provided, it defaults to CREATE_COPY:\n"
-    "[1] SKIP: Do not download these files\n"
-    "[2] OVERWRITE: Download these files and overwrite existing files\n"
-    "[3] CREATE_COPY: Download the file with a new name, appending '(1)' to the end",
+    help="The resolution method to use when a file already exists:\n"
```

Comment:
starting with resolution feels odd to me, but I guess that's what the option is named. "How to handle existing files" might be a shorter way to put it. I think it's probably fine though. Just musing out loud.